docs/running-docker.md

```diff
@@ -30,8 +29,8 @@ docker-compose up
 ### Environment variables
 
 Environment variables from your `.env` file are sourced 
-when the Python containers start
-and when you [access the running Python containers](#access-the-containers-shell). 
+when the Python container starts
+and when you [access a running Python container](#access-a-container-shell). 
```

Comment:
Similar comment about using "the" instead of "a" here, and the link target must retain the `s` in `containers` to match the slugified ID of the "Access a container's shell" heading.

```suggestion
and when you [access the running Python container](#access-a-containers-shell). 
```

Reply:
Done